app/scripts/controllers/commandLine.js

```diff
@@ -15,8 +15,36 @@ angular.module('openshiftConsole')
     $scope.loginBaseURL = DataService.openshiftAPIBaseUrl();
     $scope.sessionToken = AuthService.UserStore().getToken();
     $scope.showSessionToken = false;
+    $scope.copySessionTokenId = "showSessionTokenLink";
 
     $scope.toggleShowSessionToken = function() {
       $scope.showSessionToken = !$scope.showSessionToken;
     };
+
+    $scope.copySessionToken = function($scope) {
+      var node = $('#' + $scope.copySessionTokenId);
```

Comment:
I suggest we update the `copy-to-clipboard` directive to take in an element ID or create a new directive we could reuse (scripts/directives/util.js).